app/main-es6/plugins/hain-commands/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "hain-commands",
-  "version": "1.0.0",
-  "description": "",
+  "version": "0.0.1",
```

Comment:
I just did the same versions for all plugins.
I think it would be fine to increase the version in the inner packages after the changes (after the first release of Hain maybe).